### ISSUE TITLE
Gateway - Uniswap_v3_lp Error submitting ADD liquidity on Optimism, Arbitrum one

### ISSUE THREAD

**Issue by arnelhbot** [hummingbot/hummingbot#5642]
**Describe the bug**
Upon conducting regression testing on staging, we have noticed when starting `uniswap_v3_lp` strategy with uniswapLP_ethereum_optimism we are prompted with `Error submitting ADD liquidity order to uniswapLP on optimism for WETH-USDC` in the log pane. Docker logs traceback: Error: fractional component exceeds decimals

```
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/connector/gateway_EVM_AMM_LP.py", line 428, in _add_liquidity
    order_result: Dict[str, Any] = await self._get_gateway_instance().amm_lp_add(
  File "/home/hummingbot/hummingbot/core/gateway/gateway_http_client.py", line 679, in amm_lp_add
    return await self.api_request("post", "amm/liquidity/add", request_payload)
  File "/home/hummingbot/hummingbot/core/gateway/gateway_http_client.py", line 195, in api_request
    raise e
  File "/home/hummingbot/hummingbot/core/gateway/gateway_http_client.py", line 183, in api_request
    raise ValueError(f"Error on {method.upper()} {url} Error: {parsed_response}")
```
**Steps To Reproduce**
Here are the steps to reproduce the issue (see attachments in the section below):
1. Connect `Uniswaplp_ethereum_optimism`
2. Create `uniswap_v3_lp` strategy
3. Use `WETH-USDC` and set amount to 0.007
4. Start bot and observe `log pane`
...

**Screenshots**
![image](https://user-images.githubusercontent.com/99257257/187604973-f7c0826e-8977-452e-9edf-ab205f2e1b1c.png)

![image](https://cdn.discordapp.com/attachments/946193573311815700/1014408916840497202/unknown.png)

**Release version**
Staging 1.7.0 & Dev 1.7.0

**Attachments**
[UniswapLp_logs.zip](https://github.com/hummingbot/hummingbot/files/9458896/UniswapLp_logs.zip)

**Comment by nikspz** [comment]
Noticed same issue for arbitrum one, could be related to  Error: fractional component exceeds decimals in docker logs

**Comment by nikspz** [comment]
fix has been deployed here: https://github.com/hummingbot/hummingbot/pull/5664 and available on the Hummingbot version 1.9.0 release
